modules/backend/formwidgets/richeditor/assets/js/richeditor.js

```diff
@@ -115,6 +116,17 @@
               'oc-text-uppercase': 'Uppercase'
             }
 
+        froalaOptions.paragraphFormat = this.options.paragraphFormats
+            ? this.options.paragraphFormats
```

Comment:
Sorry, it was correct.
```suggestion
        froalaOptions.paragraphFormat = this.options.paragraphFormat
            ? this.options.paragraphFormat
```